scripts/setup.ts

```diff
@@ -10,16 +10,17 @@ const DYNAMO_LOCAL_ERROR_THREAD = `Exception in thread "main"`;
 const setupServer = (process: any) => {
   return new Promise((resolve, reject) => {
     process.stdout.setEncoding("utf-8").on("data", (stream: any) => {
+      console.log("stdout", stream);
       if (stream.includes(SERVER_OK)) {
         resolve(process);
       }
     });
 
     process.stderr.setEncoding("utf-8").on("data", (stream: any) => {
+      console.log("stderr", stream);
```

Comment:
are these console logs to be removed or are they worth keeping?

Reply:
we want to keep them, to debug the server start process